pkg/gateway/servicemap.go

```diff
@@ -10,6 +11,7 @@ type ServiceMapper interface {
 
 type ServiceMap struct {
 	ServiceGroups []ServiceGroup
+	Aliases       *Aliases
```

Comment:
Why is there an `Aliases` object in the `ServiceMap` and the `ServiceGroup`? I would only expect the latter.